e2e/commons-test.mk

```diff
@@ -66,4 +66,4 @@ functional-test: install-godog
 	ELASTIC_APM_ENVIRONMENT="${ELASTIC_APM_ENVIRONMENT}" \
 	ELASTIC_APM_SECRET_TOKEN="${APM_SECRET_TOKEN}" \
 	ELASTIC_APM_SERVER_URL="${APM_SERVER_URL}" \
-	godog --format=${FORMAT} ${FEATURES_VALUE} ${TAGS_FLAG} ${TAGS_VALUE}
+	go test -v --godog.format=${FORMAT} ${TAGS_FLAG}${TAGS_VALUE}
```

Comment:
@mdelapenya I couldnt figure out if it was possible to just pass in a single features file this way, we may need to rethink if we want to support that feature or not or just stick to tags. wdyt?